src/ansible/roles/packages/tasks/Ubuntu.yml

```diff
@@ -132,8 +140,6 @@
       - libselinux1-dev
       - libsemanage-dev
       - libsmbclient-dev
-      - libssl-dev
-      - libssl-dev
```

Comment:
OpenSSL devel is required to build SSSD. This should stay, since it is installed only conditionally for passkey.

Reply:
Added it back